packages/cli/src/commands/run.js

```diff
@@ -0,0 +1,93 @@
+import fs from 'fs'
+import path from 'path'
+
+import babelRequireHook from '@babel/register'
+import Listr from 'listr'
+import VerboseRenderer from 'listr-verbose-renderer'
+import terminalLink from 'terminal-link'
+
+import { getPaths } from 'src/lib'
+import c from 'src/lib/colors'
+
+babelRequireHook({
+  extends: path.join(getPaths().api.base, '.babelrc.js'),
+  extensions: ['.js', '.ts', '.tsx', '.jsx'],
+  only: [getPaths().api.base],
+  plugins: [
+    [
+      'babel-plugin-module-resolver',
+      {
+        alias: {
+          src: getPaths().api.src,
+        },
+      },
+    ],
+  ],
+  ignore: ['node_modules'],
+  cache: false,
+})
+
+const { db } = require(path.join(getPaths().api.lib, 'db'))
+
+const runScript = async (scriptPath) => {
+  const script = await import(scriptPath)
+  const startedAt = new Date()
+  await script.default({ db })
+  const finishedAt = new Date()
+
+  return { startedAt, finishedAt }
+}
+
+export const command = 'run'
+export const description = 'Run your script'
+export const builder = (yargs) => {
+  yargs
+  .positional('name', {
+    description: 'The name of the script to run',
+    type: 'string',
+  }).epilogue(
+    `Also see the ${terminalLink(
+      'Redwood CLI Reference',
+      'https://redwoodjs.com/docs/cli-commands#up'
+    )}`
+  )
+}
+
+export const handler = async (args) => {
```

Comment:
If you wanted to pass on flags to your script you could do something like this
```js
const {name, ...scriptArgs} =. args

/*.
.
.
 And when you call run script 
*/

await runScript(scriptPath, scriptArgs)
```

Ofcourse your runScript function will also need to pass it along :)

Reply:
Oh and also you mind need to set the `run` yargs parser to not strict

https://yargs.js.org/docs/#api-reference-strictenabledtrue

Reply:
Done. 

```
export default async ({db, args}) => {
  const [_, posArg1, posArg2] = args._
  const {k, ...kwArgs} = args
  console.log(args)
}
```

**outputs** 

```
$ /Users/viperfx/code/redwood-app/node_modules/.bin/rw run push 123 --k 23
[10:15:15] Running script [started]
{ _: [ 'run', 123 ], k: 23, '$0': 'rw' }
[10:15:18] Running script [completed]
✨  Done in 7.91s.
```